hack/jenkins/e2e.sh

```diff
@@ -497,597 +536,59 @@ case ${JOB_NAME} in
-    : ${E2E_DOWN:="false"}
-    ;;
-
-  kubernetes-upgrade-stable-latest-gke-step2-upgrade-master)
-    : ${DOGFOOD_GCLOUD:="true"}
-    : ${GKE_API_ENDPOINT:="https://test-container.sandbox.googleapis.com/"}
-    : ${E2E_CLUSTER_NAME:="gke-upgrade-stable-latest"}
-    : ${E2E_NETWORK:="gke-upgrade-stable-latest"}
-    : ${E2E_OPT:="--check_version_skew=false"}
-    # We have to get tars at ci/latest (JENKINS_PUBLISHED_VERSION default) to
-    # get the latest upgrade logic.
-    : ${JENKINS_FORCE_GET_TARS:=y}
-    : ${PROJECT:="k8s-jkns-upgrade-fixed-1"}
-    : ${E2E_UP:="false"}
-    : ${E2E_TEST:="true"}
-    : ${E2E_DOWN:="false"}
-    : ${GINKGO_TEST_ARGS:="--ginkgo.focus=Skipped.*Cluster\supgrade.*upgrade-master --upgrade-target=release/latest"}
-    ;;
-
-  kubernetes-upgrade-stable-latest-gke-step3-e2e-old)
-    : ${DOGFOOD_GCLOUD:="true"}
-    : ${GKE_API_ENDPOINT:="https://test-container.sandbox.googleapis.com/"}
-    : ${E2E_CLUSTER_NAME:="gke-upgrade-stable-latest"}
-    : ${E2E_NETWORK:="gke-upgrade-stable-latest"}
-    : ${E2E_OPT:="--check_version_skew=false"}
-    : ${JENKINS_FORCE_GET_TARS:=y}
-    # Run old e2es
-    : ${JENKINS_PUBLISHED_VERSION:="release/stable"}
-    : ${PROJECT:="k8s-jkns-upgrade-fixed-1"}
-    : ${E2E_UP:="false"}
-    : ${E2E_TEST:="true"}
-    : ${E2E_DOWN:="false"}
-    : ${GINKGO_TEST_ARGS:="--ginkgo.skip=$(join_regex_allow_empty \
-          ${GKE_DEFAULT_SKIP_TESTS[@]:+${GKE_DEFAULT_SKIP_TESTS[@]}} \
-          ${GCE_DEFAULT_SKIP_TESTS[@]:+${GCE_DEFAULT_SKIP_TESTS[@]}} \
-          ${GCE_FLAKY_TESTS[@]:+${GCE_FLAKY_TESTS[@]}} \
-          )"}
-    ;;
-
-  kubernetes-upgrade-stable-latest-gke-step4-upgrade-cluster)
-    : ${DOGFOOD_GCLOUD:="true"}
-    : ${GKE_API_ENDPOINT:="https://test-container.sandbox.googleapis.com/"}
-    : ${E2E_CLUSTER_NAME:="gke-upgrade-stable-latest"}
-    : ${E2E_NETWORK:="gke-upgrade-stable-latest"}
-    : ${E2E_OPT:="--check_version_skew=false"}
-    # We have to get tars at ci/latest (JENKINS_PUBLISHED_VERSION default) to
-    # get the latest upgrade logic.
-    : ${JENKINS_FORCE_GET_TARS:=y}
-    : ${PROJECT:="k8s-jkns-upgrade-fixed-1"}
-    : ${E2E_UP:="false"}
-    : ${E2E_TEST:="true"}
-    : ${E2E_DOWN:="false"}
-    : ${GINKGO_TEST_ARGS:="--ginkgo.focus=Skipped.*Cluster\supgrade.*upgrade-cluster --upgrade-target=release/latest"}
+    : ${JENKINS_PUBLISHED_VERSION:="ci/latest-1.1"}
+    : ${PROJECT:="k8s-jkns-e2e-gke-1-1-features"}
+    : ${FAIL_ON_GCP_RESOURCE_LEAK:="true"}
+    : ${GINKGO_TEST_ARGS:="--ginkgo.focus=GCE\sL7\sLoadBalancer\sController|Job|Horizontal\spod\sautoscaling"}
+    # At least n1-standard-2 nodes are required for the cluster to
+    # have enough cpu/ram to run the Horizontal pod autoscaling tests.
+    MINION_SIZE="n1-standard-2"
     ;;
 
-  kubernetes-upgrade-stable-latest-gke-step5-e2e-old)
-    : ${DOGFOOD_GCLOUD:="true"}
+  # Sets up the GKE soak cluster weekly using the latest CI release.
+  kubernetes-soak-weekly-deploy-gke)
```

Comment:
oh, since it didn't have -ci I assumed the soak cluster used stable, but you're totally right. good catch.

(another indication we need to clean up and be more consistent with our Jenkins job names.)